core.py

```diff
@@ -249,9 +275,12 @@ def text_style(s, style, always=False):
         :return:
         """
         # No color for pipes, files and Pythonista console
-        if not always and (isinstance(sys.stdout, StringIO)
-                           or isinstance(sys.stdout, file)
-                           or sys.stdout.write.im_self is _SYS_STDOUT):
```

Comment:
Just remembered: has anyone any idea what line 254 in `stash.py` (now `core.py`) did?
Default `sys.stdout.write` does not seem to have an `im_self` attribute and it is nowhere defined in the StaSh code, so i assumed it was a buggy line which just did not execute due to `or`-magic and commented it out.

Reply:
This is to check whether the output is Pythonista console. If it is, the value will be a StdoutCatcher object, which is only available in Pythonista.

Reply:
@ywangd thanks, i didnt check that. Should i add it back?

Reply:
Yes please.